script/push-debian.sh

```diff
@@ -43,8 +62,14 @@ if [[ -n "${ISTIO_VERSION}" ]]; then
   trap 'rm "${VERSION_FILE}"' EXIT
 fi
 
-[[ -z "${GCS_PATH}" ]] && usage
+[[ -z "${GCS_PATH}" ]] && [[ -z "${OUTPUT_DIR}" ]] && usage
 
 bazel build ${BAZEL_ARGS} ${BAZEL_TARGET}
 
-gsutil -m cp -r "${BAZEL_BINARY}.deb" ${GCS_PATH}/
+if [[ "${GCS_PATH}" != "" ]]; then
```

Comment:
The code block cares that it's set, so -n seems more appropriate than -z.  I'll use "-n" instead of "! -z" unless there's a benefit/reason for avoiding -n.